### ISSUE TITLE
Exporters : CD Release 20200219.2 - Max 2017/18/19 not available

### ISSUE THREAD

**Issue by kramsurfer** [BabylonJS/Exporters#769]
Install zip for 3ds Max 2018 is missing. 

2017 and 2019 also missing.

The new exporter installation tool is awesome! 

Thanks!

**Comment by Drigax** [comment]
Weird! that's the first time this seems to have happened. I'm creating a new build soon, I'll see if the packages fail to generate for these as well...

**Comment by Drigax** [comment]
Looks like there's something going on here. I'll continue to investigate...

**Comment by Drigax** [comment]
This feels like a bug in GitHub to be honest. 

Seems like the pipeline only started messing up sometime between 2020-02-18 and 2020-02-19, with pretty much no change to projects or our CI pipeline. I opened a support ticket with GitHub to see if they may be able to help give some context as to why this is happening, or if it something configured in our pipeline.

**Comment by kramsurfer** [comment]
Thanks!

**Comment by Drigax** [comment]
I haven't explicitly heard back from GitHub regarding why this was happening, but:
https://www.githubstatus.com/

Looking at their status for 2020-02-19 and 2020-02-25 seems to correlate to the days that we created new releases. 

Today's release appears to have been published without errors.

I think this has been resolved by GitHub, but I'll monitor the next few releases to make sure that we're successfully publishing.

Thanks for the report!
